openfl/interface/aggregator.py

```diff
@@ -91,6 +92,28 @@ def start_(plan, authorized_cols, task_group):
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
 
+        # Check if remote attestation is enabled in the plan configuration
+    if "network" in parsed_plan.config and "settings" in parsed_plan.config["network"]:
+        if parsed_plan.config["network"]["settings"].get("enable_remote_attestation", False):
+            # Fetch remote attestation environment variables
+            attestation_env = attestation_utils.fetch_attestation_env_vars()
+            if attestation_env is not None:
+                attestation_manager = attestation_utils.AttestationManager(
+                    "aggregator",
+                    attestation_env["ATTESTATION_REPORT_PATH"],
+                    attestation_env["ITA_API_KEY"],
+                    attestation_env["AVS_URL"],
+                )
+                # Generate and store the attestation report
+                attestation_manager.get_attested_identity()
+                logger.info("Remote attestation report fetched successfully.")
+            else:
+                logger.error("Failed to fetch remote attestation environment variables.")
+        else:
+            logger.info(
+                "Remote attestation is not enabled. Skipping attestation report generation."
+            )
```

Comment:
As this is practically the same code between the collaborator and aggregator, why not extract it as `get_remote_attestation`, f.e. in `attestation_utils.py` ?

Reply:
Yes - this code should be pulled out to `openfl.utils` to maintain readability of the aggregator / collaborator `start` functions